### ISSUE TITLE
Imagery button / deployment workflow improvement

### ISSUE THREAD

**Issue by bzeeb** [DMS-Aus/Roam#316]
Just thought of another helpful administrator change, could we make a new folder called _imagery as a standard folder created along with project.

When the Imagery button is hit it will load/unload any files in this folder (rasters).

Simplifies deployment as that folder will just be completely ignored by the project update, and we no longer have to copy aerials in to the server folders. Also allows site IT staff to easily swap out to the latest aerial without having to jump in to QGIS to update.

**Comment by NathanW2** [comment]
How should we handle common images between projects? Should we also have a top level _imagery folder?

**Comment by NathanW2** [comment]
Auto loading these layers is a good idea and really like that.

**Comment by Emma-D** [comment]
I have created a “shared data” folder which contains the likes of imagery, address, property and roading layers which I use to create the base for all my projects.

**Comment by NathanW2** [comment]
Yeah that is what we normally do too.  I guess having something a bit more formal is good so the application knows about it.

**Comment by NathanW2** [comment]
That does make me wonder if the Roam install folder is the best place for the `projects` folder. Should it be moved to some place else?
